gonka/versions.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "../versions.schema.json",
+  "chain_name": "gonka",
+  "versions": [
+    {
+      "name": "v0.2.0",
+      "tag": "release/v0.2.0",
+      "height": 0,
+      "recommended_version": "v0.2.0",
+      "compatible_versions": [
+        "v0.2.0"
+      ],
+      "consensus": {
+        "type": "cometbft",
+        "version": "v0.38.17",
+        "repo": "https://github.com/cometbft/cometbft"
+      },
+      "cosmwasm": {
+        "version": "v0.50.0",
+        "repo": "https://github.com/CosmWasm/wasmd",
+        "enabled": true
+      },
+      "sdk": {
+        "type": "cosmos",
+        "version": "v0.53.3-ps5",
```

Comment:
this version (with the -ps5) is not acceptable. The -ps5 can go into the tag property, but the version has to follow the standard versioning format